generators/spring-controller/index.js

```diff
@@ -58,7 +58,7 @@ module.exports = class extends BaseGenerator {
 
     get initializing() {
         if (useBlueprint) return;
-        return this._initializing();
+        return super._initializing();
```

Comment:
since you are calling `super` here we don't need `useBlueprint` check.

Reply:
can you remove the `_intializing` function above?

Reply:
You can do the same for `prompting` & `default`

Reply:
Done